docs/INSTALL.md

```diff
@@ -0,0 +1,179 @@
+<div style="display: flex; align-items: center;">
+  <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/c/c3/Python-logo-notext.svg/1200px-Python-logo-notext.svg.png" alt="Python Logo" width="100">
+  <h1 style="margin-left: 10px;">Install Python</h1>
+</div>
+
+
+
+
+A Quick Guide for Installing Python on Common Operating Systems
+
+1. [Install on Windows](#windows-)
+2. [Install on MacOS](#macos-)
+3. [Install on Linux](#linux-)
+
+<div style="display: flex; align-items: center;">
+  <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5f/Windows_logo_-_2012.svg/132px-Windows_logo_-_2012.svg.png" alt="Windows Logo" width="50">
+  <h2 style="margin-left: 10px;">Windows</h2>
+</div>
+
+
+1. If you have not yet installed Python on your Windows OS, then download and install the latest Python3 installer from [Python Downloads Page](https://www.python.org/downloads/)
+   - Make sure to check the box during installation which adds Python to PATH. Labeled something like *Add Python 3.X to PATH*
+
+2. Once Python is installed, you should be able to open a command window, type python, hit ENTER, and see a Python prompt opened. Type quit() to exit it. You should also be able to run the command pip and see its options. If both of these work, then you are ready to go.
+  - If you cannot run python or pip from a command prompt, you may need to add the Python installation directory path to the Windows PATH variable
+    - The easiest way to do this is to find the new shortcut for Python in your start menu, right-click on the shortcut, and find the folder path for the python.exe file
+      - For Python3, this will likely be something like C:\Users\<USERNAME>\AppData\Local\Programs\Python\Python37
+    - Open your Advanced System Settings window, navigate to the "Advanced" tab, and click the "Environment Variables" button
+    - Create a new system variable:
+      - Variable name: PYTHON_HOME
+      - Variable value: <your_python_installation_directory>
+    - Now modify the PATH system variable by appending the text ;%PYTHON_HOME%\;%PYTHON_HOME%;%PYTHON_HOME%\Scripts\ to the end of it.
+    - Close out your windows, open a command window and make sure you can run the commands python and pip
+### Setting up Python Environment in Windows
+
+3. **Add Python to the PATH (if necessary)**:
+   - If you cannot run `python` or `pip` from a command prompt, you may need to add the Python installation directory path to the Windows PATH variable.
+
+   - **Check Existing PATH Entry**:
+     - First, check if the `%USERPROFILE%\AppData\Roaming\Python\Python310\Scripts` path is already in your system's PATH by running this command:
```

Comment:
Inform user to pre-check if there might be some separate paths for different version and with method of installation

Reply:
some similar occurrence below too, please resolve them too.